inventory-api/src/main/scala/com/normation/inventory/domain/AgentTypes.scala

```diff
@@ -121,6 +121,19 @@ object AgentInfoSerialisation {
       info
     }
   }
+
+  def parseCompatNonJson(s: String): Box[AgentInfo] = {
```

Comment:
There should be a comment here like the one here https://github.com/Normation/ldap-inventory/pull/93/files#diff-1e32200ff4b8c1513304982443cb6e47R861
and https://github.com/Normation/ldap-inventory/pull/93/files#diff-1e32200ff4b8c1513304982443cb6e47R864 should probably use this method rather than duplicating the code

Once it's done, you can self merge